CLAUDE.md

```
@@ -687,25 +747,70 @@ runKrDailyPriceUpdate()
+{}
+```
+
+Response (201 Created)
+```json
+{
+  "guestSessionId": "uuid"
+}
+```
+
+**Set-Cookie Header:**
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)